script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -334,7 +334,7 @@ Even though I was there, I didn't witness that scene unfold in its entirety.
 [sha:4ae422443144c65e96632e1571fbef5dea369a1d]{
 -- Page 359, Offset 17468.
 -- だから言葉の上でしか想像できず、それがどれほどおぞましい暴力であるかを肌で感じ取れなかった。
-As a result I could only imagine it in words,
+As a result, I could only imagine it in words,
```

Comment:
As a result, I could only imagine the rest.

Reply:
What he's saying is that he _can't_ imagine it, he can only visualize it in terms of words. Like he can't form an image.